docs/concepts/controller-manager.md

```diff
@@ -0,0 +1,58 @@
+This is to prevent that end-users can adopt arbitrary namespaces and escalate their privileges, e.g. the `kube-system` namespace.
+
+After the namespace was created/adopted the reconciler creates several `ClusterRole`s and `ClusterRoleBinding`s that allow the project members to access related resources based on their roles.
+These RBAC resources are prefixed with `gardener.cloud:system:project{-member,-viewer}:<project-name>`.
+Gardener administrators and extension developers can define their own roles, see [this document](../extensions/project-roles.md) for more information.
+
+The `.status.phase` of the `Project` resources will be set to `Ready` or `Failed` by the reconciler to indicate whether the reconciliation loop was performed successfully.
+Also, it will generate `Event`s to provide further information about its operations.
+
+### "Stale Projects" Reconciler
+
+As Gardener is a large-scale Kubernetes as a Service it is designed for being used by a large amount of end-users.
+Over time, it is likely to happen that some of the hundreds or thousands of `Project` resources are no longer actively used.
+
+Gardener offers the "stale projects" reconciler which will take care of identifying such stale projects, marking them with a "warning", and eventually deleting them after a certain time period.
+This reconciler is enabled by default and works as following:
+
+1. Projects are considered as "stale"/not actively used when all of the following conditions apply: The namespace associated with the `Project` does not have any...
+    1. `Shoot` resources.
+    1. `Plant` resources.
+    1. `BackupEntry` resources.
+    1. `Secret` resources that are referenced by a `SecretBinding` that is in use by a `Shoot` (not necessarily in the same namespace).
+    1. `Quota` resources that are referenced by a `SecretBinding` that is in use by a `Shoot` (not necessarily in the same namespace). 
+
+If a project is considered "stale" then its `.status.staleSinceTimestamp` will be set to the time when it was first detected to be stale.
+If it gets actively used again this timestamp will be removed.
+After some time the `.status.staleAutoDeleteTimestamp` will be set to a timestamp after which Gardener will auto-delete the `Project` resource if it still is not actively used.
+
+The component configuration of the Gardener Controller Manager offers to configure the following options:
+* `minimumLifetimeDays`: Don't consider newly created `Project`s as "stale" too early to give people/end-users some time to onboard and get familiar with the system. The "stale project" reconciler won't set any timestamp for such `Project`s.
+* `staleGracePeriodDays`: Don't compute auto-delete timestamps for stale `Project`s too early to not unnecessarily make people/end-users nervous "just because" they haven't actively used their `Project` for a given amount of time.
+* `staleExpirationTimeDays`: Expiration time after which stale `Project`s are finally auto-deleted.
+
```

Comment:
Can you add an explanation about what happens if the operator changes these values for already processed projects?